packages/expo-dev-menu/ios/DevMenuManager.swift

```diff
@@ -75,6 +87,15 @@ open class DevMenuManager: NSObject, DevMenuManagerProtocol {
       }
     }
   }
+  
+  @objc
+  public static func initializeWithBridge(_ bridge: AnyObject) {
```

Comment:
After a long thought, I consider `configure(withBridge bridge: AnyObject)` would be better because
- It's more Swifty not to omit external names
- Someone may want to change the bridge later in runtime and invoking `initialize...` multiple times seems wrong and suggests you shouldn't do this, but in fact you can 😅 

```suggestion
  public static func configure(withBridge bridge: AnyObject) {
```